duneggd/Config/ArgonCube/ArgonCubeCryostat.cfg

```diff
@@ -182,49 +182,164 @@ NElements            = 0
 #=================================
 # Composite window
 #=================================
-[FaceSheetInner]
-class                = duneggd.Active.RectBar.RectBarBuilder
-Material             = 'Fiberglass'
-dx                   = 0.5*Q('7990.97mm')
-dy                   = 0.5*Q('4070.97mm')
-dz                   = 0.5*Q('25mm')
+[CompositeWindow]
+class = duneggd.ArgonCube.dune_window.WorldBuilder
+subbuilders = ["block", "smaller_block", "support_beam", "sideways_support_beam","middle_triangle_holes","side_triangle_holes", "zaxis_smaller_block", "support_beam_end_hole","support_beam_end_line","intersection_beam_line", "support_beam_end_hole_hori", "inner_sheet", "outer_sheet", "middle_triangle_holes_hori", "side_triangle_holes_hori" ]
+material = "Air"
+show_cover_sheets = True #covers the front and back, set to false to see the window design inside
+verti_active_area = Q("3285mm") / 2 #uses 3285 not 3030 from diagram, 3030 breaks this.
+hori_active_area = Q("7000mm") / 2
+dx = Q("8620mm") * 1.001 / 2
+dy = Q("3800mm") * 1.001 / 2
+dz = ( Q("403.19mm")   ) / 2
 
-[Core]
-class                = duneggd.Active.RectBar.RectBarBuilder
-Material             = 'Polyurethane' 
-dx                   = {FaceSheetInner:dx} 
-dy                   = {FaceSheetInner:dy} 
-dz                   = 0.5*Q('343.19mm')
 
-[FaceSheetOuter]
+[CompositeWindowMembrane]
 class                = duneggd.Active.RectBar.RectBarBuilder
-Material             = {FaceSheetInner:Material} 
-dx                   = {FaceSheetInner:dx} 
-dy                   = {FaceSheetInner:dy} 
-dz                   = {FaceSheetInner:dz} 
+Material             = 'Plywood'
```

Comment:
The Changelog said this should be steel. Or is it talking about a different membrane?